code/segmentedtickerplant/stplog.q

```diff
@@ -260,8 +274,8 @@ init:{[dbname]
   getnextendUTC[]; 
   i::1; // default value for log seq number
 
-  if[value `..createlogs;
-    createdld[.proc.procname;.eodtime.d];
+  if[(value `..createlogs) or .sctp.sctploggingmode=`create;
+    createdld[dbname;.eodtime.d];
```

Comment:
dbname is the name of the function parameter here so whatever the input is will be the name. I was avoiding hardcoding it because the tests create unique test logs with different names
in a normal initialisation the procname is the function input